sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,15 +1,19 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.2 (2022-03-14)
```

Comment:
Should us use 1.1.0? This will be the first GA version that supports push notification in Android SDK. @juancamilor

Reply:
Oh no, this is definitely not that. We're not making all the new functionality GA. The only difference between this release and `1.0.1` are dependency versions.